### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by JuliePrazakova**
waiting for fix in the component types
https://github.com/RedHatInsights/frontend-components/pull/1635

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,47 +1,30 @@
...
-  );
+  useEffect(() => {
+    fetch(`/api/quickstarts/v1/quickstarts?bundle=settings`)
+      .then<{ data: { content: QuickStart }[] }>((response) => response.json())
+      .then((response) =>
+        quickStarts.set(
+          'settings',
+          response.data.map((i) => i.content)
+        )
+      )
```
Comment (karelhala) [inline src/App.tsx:21]: You can use destruct in here
```suggestion
      .then(({ data }) =>
        quickStarts.set(
          'settings',
          data.map(({ content }) => content)
        )
      )
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,47 +1,30 @@
...
+      .catch((err) => {
+        console.log(err.message);
+      });
+  }, []);
```
Comment (karelhala) [inline src/App.tsx:25]: I think we should watch for `quickStarts` in here and I'd also run the fetch just when `quickStarts` is defined.

File: src/App.tsx [hunk 3]
```diff
@@ -1,47 +1,30 @@
...
+      .then<{ data: { content: QuickStart }[] }>((response) => response.json())
+      .then((response) =>
+        quickStarts.set(
+          'settings',
+          response.data.map((i) => i.content)
+        )
+      )
+      .catch((err) => {
+        console.log(err.message);
+      });
```
Comment (karelhala) [inline src/App.tsx:24]: As a proof of concept this is fine, I would much rather see a proper integration to notifications middleware and show the error why it failed.

**Review by karelhala — COMMENTED** [review]
Just a few questions, but overall looks good!

File: src/App.tsx (hunk 3 above)
Comment (Hyperkid123) [reply src/App.tsx:24]: Lets keep this one for a follow up PR :slightly_smiling_face:

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**